### PR TITLE
📦 NEW: Add Donate link mrahmadawais & ahmadawais

### PR DESCRIPTION
**Pull request by ahmadawais**
This PR adds donate links for my packages

Thanks for creating this awesome tool.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ const authors = {
   mmckegg: 'https://www.patreon.com/MattMcKegg',
   moox: 'https://liberapay.com/MoOx/',
   mpj: 'https://www.patreon.com/funfunfunction',
+  mrahmadawais: 'https://pay.paddle.com/checkout/515568',
+  ahmadawais: 'https://pay.paddle.com/checkout/515568',
```
Comment (feross) [inline index.js]: This is not a valid npm user: https://www.npmjs.com/~ahmadawais

Please remove this line

File: index.js [hunk 2]
```diff
@@ -75,6 +77,7 @@ const packages = {
   'axboot': 'https://opencollective.com/ax-boot-framework',
   'babel': 'https://opencollective.com/babel',
   'babel-core': 'https://opencollective.com/babel',
+  'babel-preset-cgb': 'https://pay.paddle.com/checkout/515568',
```
Comment (feross) [inline index.js]: Please remove all the individual packages from this section. It's sufficient to add yourself to the `authors` object for all your packages to be detected.

**Review by feross — CHANGES_REQUESTED** [review]
Thanks for the PR. There are some issues you need to fix before I can merge this.

**Comment by ahmadawais** [comment]
@feross Sure, I just fixed all of that. Only the author and link names.

```js
mrahmadawais: 'https://pay.paddle.com/checkout/515568',
```

Looking forward!

**Review by feross — APPROVED** [review]

**Comment by feross** [comment]
Thanks @ahmadawais!

**Pull request merged**